src/core/Codend.Application/Projects/Commands/CreateProject/CreateProjectCommand.cs

```diff
@@ -67,18 +68,29 @@ public async Task<Result<Guid>> Handle(CreateProjectCommand request, Cancellatio
             return result.ToResult();
         }
 
+        // Set To-Do as first project default status.
+        var defaultStatus = resultStatuses.FirstOrDefault(status => 
+            status.Value.Name.Value == nameof(DefaultTaskStatus.ToDo))?.Value;
```

Comment:
idk does it even work?

Reply:
it does xD, if you have better idea just rewrite.